enJsonCounter.py

```diff
@@ -13,195 +14,185 @@ def main():
     import re
     import collections
     import datetime
+    from contextlib import contextmanager
 
     #class imports
-    from classes.steps import Step3
-    from classes.steps import Step1and2
+    #from classes.steps import Step3
+    #from classes.steps import Step1and2
```

Comment:
Should these be commented?

Reply:
Absolutely, sounds like a plan. Thank you for the feedback.